install_template/README.md

```diff
@@ -225,3 +225,44 @@ In a handful of situations it is useful to employ conditionals to include or mod
 Be wary of over-using these; prefer inheritance structures that push this content down to sufficiently-specific "leaf" templates instead. As a general rule, use conditionals only when the test is much, much simpler than the alternative inheritance structure, and be willing to abandon it when (over time) that simplicity is lost.
 
 In particular, avoid the trap of setting a flag in a leaf template and then checking it in a base template: this separates the context from the decision, making it extremely difficult to judge when the test has lost its value.
+
+#### Adding a new platform
+
+1. Modify **config.yaml**. For example, when adding the RHEL 9 platform, the following entries were made to each product:
+
+- name: RHEL 9
+  arch: ppc64le
+  supported versions: [<supported_versions>]
+- name: AlmaLinux 9 or Rocky Linux 9
+   arch: x86_64
+   supported versions: [<supported_versions>]
+- name: RHEL 9 or OL 9
+   arch: x86_64
+   supported versions: [<supported_versions>]
+   
+1. In **templates/platformBase_deploymentConstants.njk**, update the `map_platform` and `map_platform_old` blocks. For example, for RHEL 9, the following lines were added to both blocks of code:
+
+"AlmaLinux 9 or Rocky Linux 9": "other_linux9",
```

Comment:
What's the difference between other_linux9 and rhel9? Again, where do these names come from?

Reply:
This will be a fun discussion point for you and Nidhi.